ssc/cmod_fuelcell.cpp

```diff
@@ -137,13 +137,15 @@ void cm_fuelcell::exec()
 {
 	double annual_energy = 0.0;
 	double annual_fuel = 0.0;
-	float percent_complete = 0.0;
+	// float percent_complete = 0.0;
```

Comment:
Lines 34 and 211 in this file should also be commented out if this is commented out. Could you please add those?